crates/services/relayer/src/config.rs

```diff
@@ -22,8 +22,8 @@ pub(crate) static ETH_FORCED_TX: Lazy<H256> =
 pub struct Config {
     /// The da block to which the contract was deployed.
     pub da_deploy_height: DaBlockHeight,
-    /// Uri address to ethereum client.
-    pub relayer: Option<url::Url>,
+    /// Uri addresses to ethereum client.
+    pub relayer: Option<Vec<url::Url>>,
```

Comment:
```suggestion
    pub relayer: Vec<url::Url>,
```

Reply:
Can we rename this to `relayers`, or would that entail a lot of downstream work since we'd break potential existing configurations?

Reply:
This command should probably should've been called something like `eth-rpc-urls` since theres only one relayer running inside of fuel-core. But it's a breaking change that would impact a lot of people.

Reply:
Thanks, yeah that's what I was afraid of. A non breaking variant would be to add another arg with the intended name and mark this one as deprecated in its docstring. But I suppose we can await doing that until we have more fields to rename.